ci/slab.toml

```diff
@@ -94,6 +94,11 @@ workflow = "concrete_python_tests_linux.yml"
 profile = "m6i-cpu-test"
 check_run_name = "Concrete Python Tests (Linux)"
 
+[command.concrete-python-tests-linux-gpu]
+workflow = "concrete_python_tests_linux.yml"
+profile = "gpu-test"
```

Comment:
Could you maybe try with the gpu-bench profile? I think the gpu-test profile has a setup with docker for the compiler, but in your case you're just recompiling everything from scratch no?